site/docs/spec.md

```diff
@@ -489,7 +489,7 @@ Lists must use the [3-level representation](https://github.com/apache/parquet-fo
 
 One of the interesting challenges with this is how to map Iceberg’s schema evolution (id based) on to ORC’s (name based). In theory, we could use Iceberg’s column ids as the column and field names, but that would suck from a user’s point of view. 
 
-The column ids would be stored in ORC’s user metadata as `iceberg.column.id` with a comma separated list of the ids.
+The column IDs must be stored in ORC type attributes using the key `iceberg.id`, and `iceberg.required` to store `"true"` if the Iceberg column is required, otherwise it will be optional.
```

Comment:
I think this is fine. There's no need to use more ID names than necessary. Avro uses several properties because it can't store properties with every type.

Reply:
> `iceberg.required` to store `"true"` if the Iceberg column is required
Do we have a counterpart of this in Parquet?

Reply:
Yes. Parquet fields have a repetition level that is REQUIRED, OPTIONAL, or REPEATED. REPEATED is used for collections (list and map). REQUIRED and OPTIONAL are used to store this for fields.